.gitignore

```diff
@@ -1,9 +1,6 @@
 # Ignore distributables folder
 dist/
 
-# Ignore vscode folder
-.vscode/
-
```

Comment:
No need to remove the ignore to the vscode server. We can ignore it and add the changes at the same time.
For example I have some logs in the folder that are autogenerated and they pollute the `git status` command.

![image](https://user-images.githubusercontent.com/46280801/218682605-cd56812f-ab7a-4620-8eed-855d7ff24908.png)

If we don't ignore it, there might also be a change to add these files accidentally.
I say we ignore it, but keep the `settings.json` change.